modules/@ergo/core/components/Inputs/Select/Select.vue

```diff
@@ -368,6 +359,15 @@ export default {
         window.removeEventListener('click', this.onClickOutside);
     },
     methods: {
+        getDropDownOffset() {
+            const {
+                x, y, width, height,
+            } = this.$refs.activator.getBoundingClientRect();
+
+            return {
+                x, y, width, height,
+            };
+        },
```

Comment:
this function repeats in several places:
ActionBaseButton.vue 
GridAdvancedFilter.vue 
MainSettingsForm.vue

Reply:
I agree, but the only way to move it to one place is by creating mixin. It would be not readable - you might have different ref of component from which you wanna get an offset. It might confuse more. I'll think about it how better we could handle it - maybe by extracting more logic into sth? I do not know.